.github/workflows/backend.yml

```diff
@@ -63,3 +77,17 @@ jobs:
                  -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
       - name: Check dependency license
         run: tools/dependencies/check-LICENSE.sh
+  result:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    needs: [ build ]
```

Comment:
Did we miss job `paths-filter` dependent here?

Reply:
It does not need to. It already exists in steps.run.

Reply:
@zhongjiajie we don't need explicit dependent here, `result` depends on `build`, `build` depends on `paths-filter`, so `result` depends on `paths-filter` implicitly

Reply:
Thanks for clarification.